features/step_definitions/preferences_steps.rb

```diff
@@ -74,4 +74,10 @@
   user = User.current_user
   user.preference.history_enabled = false
   user.preference.save
-end
\ No newline at end of file
+end
+
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.